modules/build/src/main/scala/scala/build/errors/Severity.scala

```diff
@@ -0,0 +1,8 @@
+package scala.build.errors
+
+sealed trait Severity
+
+object Severity {
+  object ERROR   extends Severity
+  object WARNING extends Severity
```

Comment:
NIT See above, and we tend not to use upper case in class names or enums, more camel case.
```suggestion
  case object Error   extends Severity
  case object Warning extends Severity
```